01/app.js

```diff
@@ -1 +1,12 @@
-console.log('DOM');
\ No newline at end of file
+console.log('DOM');
+let comment, dataInfo;
+
+const commentsList = document.querySelector(`.comments`);
+
+if (commentsList === null) {
```

Comment:
Można też 
```
if(commentsList) {
  // ...
} else {
  console.log(`Something goes wrong...`);
}
```
Kolejność też jest o tyle istotna, że na początku jest to co czego się spodziewamy - to co ma zostać wykonane. W mojej ocenie jest to czytelniejsze ;)